### PR TITLE
Dm reply

### PR DESCRIPTION
**Pull request by bob2402**
image

### CODE REVIEW

File: app/nostr.ts [hunk 1]
```diff
@@ -141,6 +141,45 @@ export async function prepareNostrImageEvent(
     return signedEvent;
 }
 
+export async function prepareReplyEncryptNostrEvent(
+    sender: nostr.NostrAccountContext,
```
Comment (BlowaterNostr) [inline app/nostr.ts:145]: With the understanding of NIP-42 that other people/agent can repost/broadcast your events, "sender" is not the accurate word to describe it. `author` or `signer` is a better word.

File: app/nostr.ts [hunk 2]
```diff
@@ -141,6 +141,45 @@ export async function prepareNostrImageEvent(
...
+                ...tags,
+            ]
+            : [
+                [
+                    "e",
+                    targetEvent.id,
+                    currentRelay,
+                    "reply",
+                ],
+            ],
```
Comment (BlowaterNostr) [inline app/nostr.ts:178]:
```js
[
                [
                    "e",
                    targetEvent.id,
                    currentRelay,
                    "reply",
                ],
                ...tags,
            ]
```
should be enough

**Review by BlowaterNostr — CHANGES_REQUESTED** [review]

File: app/nostr.ts (hunk 1 above)
Comment (bob2402) [reply app/nostr.ts]: ok, i use `signer`

File: app/nostr.ts (hunk 2 above)
Comment (bob2402) [reply app/nostr.ts:167]: ok

**Review by BlowaterNostr — CHANGES_REQUESTED** [review]
Things to add:
1. editor.test.tsx
Let's fix the color and have this test itself able to switch between reply mode and normal mode

Let's address this issue first before proceeding

**Comment by bob2402** [comment]
> Things to add:
> 
> 1. editor.test.tsx
>    Let's fix the color and have this test itself able to switch between reply mode and normal mode
> 
> Let's address this issue first before proceeding

Okay, I have fixed the editor style and completed the test.

**Pull request merged**